scene/main/scene_tree.cpp

```diff
@@ -1408,7 +1408,7 @@ SceneTree::SceneTree() {
 	//create with mainloop
 
 	root = memnew(Window);
-	root->set_name("root");
+	root->set_name("Root");
```

Comment:
Won't that break compat with scripts that access `get_node("/root")`?

Reply:
I'm quite sure everyone just uses `get_tree().root`, as it makes more sense.

Reply:
I've seen `get_node("/root")` in the wild quite often, so if we change this, we should make this prominent in the changelog.

Reply:
That's weird that people would go for that route, but if you say so.
The main reason for this change is to make it match the other node names, as everything follows PascalCase.

Reply:
I'd leave that out for this PR, this simple change is worth a GIP. :)
The rest is good to merge.